src/services/messenger.py

```diff
@@ -247,8 +247,8 @@ def process_postback(messenger, payload):
         qrs = quick_replies.QuickReplies(quick_replies=[qr])
         text = {
             "text": _(
-                u'👩🏽‍🏫 You know ...\n✔️ Wordpress,\n✔️ Notepad++,\n✔️ Ubuntu\n'
-                'and thousand of common software started out as open source'
+                u'👩🏽‍🏫 Did you know ...\n✔️ Wordpress,\n✔️ Notepad++,\n✔️ Ubuntu\n'
```

Comment:
This line have more than 80 characters and will not pass the flake8 test, break it in 2!